src/Views/FirmwareView.vala

```diff
@@ -240,7 +240,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
     }
 
     private async void update (Fwupd.Device device, Fwupd.Release release) {
-        progress_alert_view.title = _("“%s” is being updated").printf (device.get_name ());
+        progress_alert_view.title = _("“%s” is getting new firmware…").printf (device.get_name ());
```

Comment:
This sounds much less natural

Reply:
It also explains the process of what is actually happening.